src/pages/Home/components/SupplyRank/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Chip } from '@/components';
+import { useSupplyRank } from '@/hooks/useSupplyRank/useSupplyRank';
+import { Loader } from 'lucide-react';
+
+export const SupplyRank = () => {
+  const { data: supplyRankRows, loading } = useSupplyRank();
+
+  return (
+    <div className="my-2">
+      <h2 className="text-[#2f2f2f] font-semibold text-xl">
```

Comment:
```suggestion
      <h2 className="text-[#2f2f2f] font-semibold text-xl pb-2">
```

Complementando o comentário do https://github.com/SOS-RS/frontend/pull/67#pullrequestreview-2050573197